tests/test_data/pwdb_fragments/4c4917157d621114c4d108fb230ad337650d7c07a786ef914bae62c4c9e8ccd7.json

```diff
@@ -0,0 +1,78 @@
+{
+  "identifier": "95",
```

Comment:
This file must be in gitignore or not ?

Reply:
nop, this file is used for tests as well